sdk/eventgrid/azure-eventgrid/azure/eventgrid/shared_access_signature_credential.py

```diff
@@ -0,0 +1,32 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See LICENSE.txt in the project root for
+# license information.
+# -------------------------------------------------------------------------
+
+class EventGridSharedAccessSignatureCredential(object):
```

Comment:
Is this file expected to be public?

Reply:
Yes, similar to AzureKeyCredential, so that it can be passed in as the credential to EventGridPublisherClient